webextension/js/confirm-page.js

```diff
@@ -0,0 +1,25 @@
+const redirectUrl = new URL(window.location).searchParams.get("url");
+document.getElementById("redirect-url").textContent = redirectUrl;
+const redirectOrigin = new URL(redirectUrl).origin;
```

Comment:
`const redirectHostname = new URL(redirectUrl).hostname` and `textContent = redirectHostname` will probably look cleaner?